uniffi/src/bindings/kotlin/templates/RustBufferHelpers.kt

```diff
@@ -141,6 +141,37 @@ fun Double.lowerInto(buf: ByteBuffer) {
     buf.putDouble(this)
 }
 
+fun String.lower(): String {
+    return this
+}
+
+fun String.lowerInto(buf: ByteBuffer) {
+    val byteArr = this.toByteArray()
+    buf.putInt(byteArr.size)
+    buf.put(byteArr)
+}
+
+fun String.Companion.liftFrom(buf: ByteBuffer): String {
+    val len = Int.liftFrom(buf)
+    val byteArr = ByteArray(len)
+    buf.get(byteArr)
+    return byteArr.toString(Charsets.UTF_8)
+}
+
+fun String.lowersIntoSize(): Int {
+    return this.toByteArray().size + 4
```

Comment:
It's a shame we have to `toByteArray` the string twice when lowering into a buffer (once to pre-calculate the size, and again when actually writing it into the buffer). But that can be a problem for a future refactoring.